src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -92,9 +93,9 @@ public void parseCommand_list() throws Exception {
 
     @Test
     public void parseCommand_remark() throws Exception {
```

Comment:
Possible to change the naming from remark 😆

Reply:
You can do shift-shift to search all the files for any traces of remark left, and change it to notes!